docs/api/resourcemanager.md

```diff
@@ -1,5 +1,13 @@
 # API Reference
 
+**Important:**
+
+> As of this release, all vendor-related resources—including Vendor, VendorTypeDefinition, VendorVerification, and CorporationTypeConfig—have been moved from the `resourcemanager.miloapis.com/v1alpha1` API group to the new `vendors.miloapis.com/v1alpha1` API group.
+>
+> This document *only* covers resource types that remain in the `resourcemanager.miloapis.com/v1alpha1` API group (such as Organization, OrganizationMembership, and Project).
+>
+> For vendor management and vendor type APIs, see the [vendors.miloapis.com API documentation](./vendors-api-group.md).
+
```

Comment:
I'm confused by this section of the documentation. Was this something that AI added after moving the vendors stuff to a new API type? This document is automatically generated by `task generate`.

Reply:
Removed!